testsuite/MDAnalysisTests/coordinates/test_xyz.py

```diff
@@ -16,7 +16,7 @@
 from MDAnalysisTests.coordinates.base import (BaseReaderTest, BaseReference,
                                               BaseWriterTest)
 from MDAnalysisTests import tempdir, make_Universe
-
+from numpy.testing import TestCase
```

Comment:
Looks like it - https://github.com/numpy/numpy/blob/master/numpy/testing/__init__.py#L10
Should be able to just use `unittest.TestCase` in its place.